scripts/preprocess_smithsonian.py

```diff
@@ -0,0 +1,60 @@
+import click
```

Comment:
This was used to generate the temp assets we use to run the importer.  If it belongs somewhere else, let me know!